Assets/Scripts/Localization/LocalizationLoader.cs

```diff
@@ -33,19 +33,27 @@ void Awake()
 
             //Get the filepath.
             string filePath = Path.Combine(Application.streamingAssetsPath, "Localization");
-            
+
             //Loop through all files.
-            foreach(string file in Directory.GetFiles(filePath))
+            //TODO: Think over the extension ".lang", might change that in the future.
+            foreach (string file in Directory.GetFiles(filePath, "*.lang"))
+            {
+                //The file extention is .lang, load it.
+                LocalizationTable.LoadLocalizationFile(file);
+
+                //Just write a little debug info into the console.
+                Logger.Log("Loaded localization at path\n" + file);
+            }
+
+            foreach (DirectoryInfo mod in WorldController.Instance.ModsManager.GetMods())
             {
-                //Check if the file is really a .lang file, and nothing else.
-                //TODO: Think over the extension ".lang", might change that in the future.
-                if(file.EndsWith(".lang"))
+                foreach (FileInfo file in mod.GetFiles("*.lang"))
                 {
-                    //The file extention is .lang, load it.
-                    LocalizationTable.LoadLocalizationFile(file);
+                    //Load the mod localization.
+                    LocalizationTable.LoadLocalizationFile(file.FullName);
 
                     //Just write a little debug info into the console.
-                    Logger.Log("Loaded localization at path\n" + file);
+                    Logger.Log("Loaded mod localization at path\n" + file);
```

Comment:
This log thats within 2 loops looks like it'll be spam problem later on. Make it a Verbose Log ?